packages/terra-consumer-nav/src/components/nav-help/NavHelpPopup.scss

```diff
@@ -11,7 +11,7 @@
   .close-button {
     background: none;
     border: 0;
-    color: var(--terra-consumer--popup-close-button-color, #fff);
+    color: var(--terra-consumer-close-button-color, #fff);
```

Comment:
This style is specific to the close button in popup header. If we use --terra-consumer-close-button-color then it seems all the close buttons would use the same color. Should we follow the similar pattern used above in Modal.scss  **--terra-consumer-close-button-color** ?

Reply:
Yes, since this is specific to Popup I will change it to `--terra-consumer-popup-close-button-color`. Similarly, hover and focus will change to `--terra-consumer-popup-close-button-hover-color` and `--terra-consumer-popup-close-button-focus-color` respectively.